api/GetRainfallByAddress/__init__.py

```diff
@@ -46,6 +66,9 @@ def main(req: func.HttpRequest) -> func.HttpResponse:
     monthly_consumption = get_water_usage("21270260")
     # TODO: implement check to ensure monthly data returns same # of periods
 
+    # Water savings
+    predicted_savings_potential, water_harvest_potential = potential_savings(monthly_consumption, annual_rain, roof_surface_area_in_square_meters)
```

Comment:
We don't use `water_harvest_potential` when returning right? It could replace:
`"AnnualRainCollectionMm": round(annual_rain * roof_surface_area_in_square_meters, 2),`
to
`"AnnualRainCollectionMm": water_harvest_potential` right?

Reply:
Yep we can